marimo/_mcp/__init__.py

```diff
@@ -0,0 +1,8 @@
+# Copyright 2024 Marimo. All rights reserved.
+
+from __future__ import annotations
+
+from marimo._mcp.registry import MCPRegistry
+from marimo._mcp.server import MCPServer
```

Comment:
since we already namespace this under `mcp`, maybe:

```
from marimo._mcp.registry import MCPRegistry as Registry
from marimo._mcp.server import MCPServer as Server
```

so then downstream, it reads as `mo.mcp.Server`